templates/cli/Dockerfile.twig

```diff
@@ -0,0 +1,18 @@
+FROM composer:2.0 as step0
```

Comment:
Using composer image as a production image isn't considered a best practice. We should use Docker multistage builds. You can learn more about it here: https://docs.docker.com/develop/develop-images/multistage-build/ and check the Appwrite dockerfile as a reference.

Reply:
As discussed, I have made use of multi-stage build